vault/logical_system.go

```diff
@@ -4791,7 +4778,6 @@ func (core *Core) GetSealStatus(ctx context.Context) (*SealStatusResponse, error
 		ClusterID:    clusterID,
 		RecoverySeal: core.SealAccess().RecoveryKeySupported(),
 		StorageType:  core.StorageType(),
-		Warnings:     core.getStatusWarnings(),
```

Comment:
nit: This Warnings seems to be used only for SSCT. It make sense to me to keep it around, but I was wondering if the corresponding struct should also be modified?

Reply:
Modified how?

Reply:
Like to remove the warnings all together.

Reply:
Ok, but I'm not sure how to reconcile that with your saying "it makes sense to me to keep it around".  It made sense to me as well, we might want to add new warnings in future.

Reply:
Sorry for the confusion. I just wanted to learn your thought process on keeping that field around which as you mentioned is for potential future use.